src/providers/Mparticle.js

```diff
@@ -10,7 +10,7 @@ class MparticleProvider extends BaseProvider {
     constructor() {
         super();
         this._key       = "MPARTICLE";
-        this._pattern   = /\.mparticle\.com\/v\d\/JS\/[^/]+\/Events$/;
+        this._pattern   = /\.mparticle\.com\/v\d\/JS\/[^/]+\/events/i;
```

Comment:
Hmm, we may need to rethink this part - either changing it to `[eE]` or having _all_ of the patterns be case-insensitive.

per https://github.com/MisterPhilip/omnibug/blob/master/src/providers/OmnibugProvider.js#L39, we have:
```js
let x = new RegExp(defaultPattern.map((el) => {
                return el.source;
            }).join("|"));
```

So once we run that in context of multiple providers:
```js
let defaultPattern = [/tESt/i, /fooBAR/];
let x = new RegExp(defaultPattern.map((el) => {
                return el.source;
            }).join("|"));
// x is now /tESt|fooBAR/ - note the missing case insensitive flag for tESt

x.test("test"); // returns false
```

Reply:
I think changing it to just [eE] is ok for now. Then revisit the bigger conversation when doing a real provider upgrade for v3. I will make this change and resubmit

Reply:
No need for a full resubmit, just commit a new change and it'll add it on to the pull request 😄

Reply:
done